common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -7,6 +7,23 @@ import scala.util.Try
 
 import com.gu.mediaservice.model.{FileMetadata, ImageMetadata}
 
+object SupplierCategory extends Enumeration {
+  type SupplierCategory = Value
+  val Sport = Value("Sport")
+  val Entertainment = Value("Entertainment")
+  val Unknown = Value("Unknown")
+
+  // These category codes are now deprecated but still populated
+  // See: https://www.iptc.org/std/photometadata/documentation/GenericGuidelines/index.htm#!Documents/guidelineformappingcategorycodestosubjectnewscodes.htm
+  def create(category: String) = category match {
```

Comment:
Shouldn't we have loads more mappings in here?

Only raising now because it's quite an expensive process to map them.